using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -126,6 +118,75 @@ Set the Push Identifier request
 }
 ```
 
+### Analytics Identity Response
+
+Identity Extension listens to Analytics Identity Response Event to retrieve the value of the Analytics tracking identifier (AID) that is returned as part of the visitor information for the  appendToURL and getUrlVariables API calls. 
+
+For more details see [Analytics response identity Document](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-event-reference#analytics-response-identity).
+
+### Audience Manager Content Response
+
+The Identity extension listens for the Audience extension response content event for any user opt-out updates. If this response contains `optedouthitsent: false` an opt-out network call is sent to the Experience Cloud Identity Service.
+
+For more details see [Audience Manager Content Response Document](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-audience-manager/audience-manager-event-reference#audience-manager-content-response-1).
+
+### Configuration Response Content
+
+#### Event Description
+
+Whenever the configuration extension dispatches the Configuration Response Content event, the Identity extension fetches the URL of the Experience Cloud Identity Service,  Experience Cloud Organization Identifier, and the Privacy status settings.
+
+This event will be generated in the following scenarios:
```

Comment:
Use present tense for all the sentenses. E.g. This event is generated in the following scenarios: